README.md

```diff
@@ -1,57 +1,27 @@
 # Northcoders House of Games API
 
-## Background
+Once you have cloned down the project, run
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+`npm install`
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+to install all the relevant packages.
+Most of what is needed is there already, but you will additionally need to run
+`npm install express`
 
-## Kanban
+and
+`npm install supertest`
```

Comment:
The user wont have to explicity install these as long as you have installed them at somepoint since they will be in the package.json and when we run `npm install`, all the dependencies in the package.json will be installed for them